src/components/Conversation/EmptyMessageFeed.tsx

```diff
@@ -87,6 +95,40 @@ export default function EmptyMessageFeed({ onAddMemberClick }: Props) {
     )
 }
 
+function TokenGatedMessage() {
+    const { tokenAddress, tokenIds, tokenType, stakingEnabled } = useSelectedRoom() || {}
+
+    if (!tokenType || tokenIds) {
+        return null
+    }
+    const tokenMetadata = useTokenMetadata(tokenAddress, tokenIds || [])
+    const polygonScanUrl = `https://polygonscan.com/token/${tokenAddress}`
+
+    const displayTokenAddress = useDisplayUsername(tokenAddress, {
```

Comment:
Is it possible (likely) that a token address has an alias (within the app) or an ENS domain assigned to it?

Reply:
doesn't `useDisplayUsername` already check for ENS names?